nautobot/extras/management/__init__.py

```diff
@@ -77,25 +114,24 @@ def export_statuses_from_choiceset(choiceset, color_map=None, description_map=No
     choices = []
 
     for slug, value in choiceset.CHOICES:
-        css_class = choiceset.CSS_CLASSES[slug]
-
         choice_kwargs = dict(
             name=value,
             slug=slug,
             description=description_map[slug],
-            color=color_map[css_class],
+            color=color_map[slug],
         )
         choices.append(choice_kwargs)
 
     return choices
 
 
 def create_custom_statuses(
-    app_config,
+    app_config=None,
     verbosity=2,
     interactive=True,
     using=DEFAULT_DB_ALIAS,
     apps=global_apps,
+    models=list(CHOICESET_MAP.keys()),
```

Comment:
Because this is a mutable mapping, even tho it's not being mutated, having it in a function signature should be avoided.

Please move this to a sentinel value check in the function body a la:

```python
    if models is None:
        list(CHOICESET_MAP)
```

```suggestion
    models=None,
```